ansible_roles/init/defaults/main.yml

```diff
@@ -0,0 +1 @@
+key_path: integration-tests/config/leappto_testing_key.pub
```

Comment:
Yeah, it's quirky, I'll probably change it to an empty string just so that the variable is listed in the defaults file.